app/controllers/apo_controller.rb

```diff
@@ -19,6 +19,38 @@ def new
     render layout: 'one_column'
   end
 
+  # Draw the form controls for collection, rights and initial workflow
+  def registration_options
```

Comment:
This method consists of code copied from the three methods previously part of RegistrationController https://github.com/sul-dlss/argo/pull/3572/files#diff-99d7bf3698ba31b7aedf0f330940825d7f298fe8a94391d8e08281c8c64078d1L17-L77